### PR TITLE
Update manage.py

### PR DESCRIPTION
**Pull request by youssef7ussien**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -23,5 +23,5 @@ def main():
     execute_from_command_line(sys.argv)
 
 
-if __name__ == '__main__':
-    main()
+
+main()
```
Comment (ahmedsayed19) [inline manage.py:27]: test comment: That's not clean code!

File: manage.py [hunk 2]
```diff
@@ -23,5 +23,5 @@ def main():
     execute_from_command_line(sys.argv)
```
Comment (ahmedsayed19) [inline manage.py:23]: Test comment: update this

File: manage.py [hunk 3]
```diff
@@ -23,5 +23,5 @@ def main():
...
 
 
-if __name__ == '__main__':
```
Comment (ahmedsayed19) [inline manage.py:26]: test comment: don't remove this line

**Review by ahmedsayed19 — CHANGES_REQUESTED** [review]
Great project submission but it needs some work

File: manage.py (hunk 1 above)
Comment (youssef7ussien) [reply manage.py:27]: Why my bro is so clean and beautiful

File: manage.py (hunk 3 above)
Comment (youssef7ussien) [reply manage.py:26]: Why my bro is so clean and beautiful

File: manage.py (hunk 2 above)
Comment (youssef7ussien) [reply manage.py:23]: Why my bro is so clean and beautiful

**Review by youssef7ussien — COMMENTED** [review]
Why my bro is so clean and beautiful

**Pull request closed without merging**